src/panels/config/automation/action/types/ha-automation-action-choose.ts

```diff
@@ -22,6 +44,8 @@ import { fullEntitiesContext } from "../../../../../data/context";
 import { EntityRegistryEntry } from "../../../../../data/entity_registry";
 import { sortableStyles } from "../../../../../resources/ha-sortable-style";
 
+const preventDefault = (ev) => ev.preventDefault();
```

Comment:
We should move this to `src/common/dom/prevent-default.ts` like we do for `stopPropagation`